Samples/Hast.Samples.SampleAssembly/ImageSharpSample.cs

```diff
@@ -0,0 +1,101 @@
+using Hast.Synthesis.Abstractions;
+using Hast.Transformer.Abstractions.SimpleMemory;
+using System.Threading.Tasks;
+
+namespace Hast.Samples.SampleAssembly;
+
+/// <summary>
+/// Algorithm for resizing images with a modified ImageSharp library.
+/// </summary>
+public class ImageSharpSample
```

Comment:
Ez nem csak egy sample, hanem a Hastlayer-accelerated ImageSharp része.

Reply:
Itt mi történt? Látom kapott 👍-t viszont jelenleg továbbra is `ImageSharpSample`. De szerintem ez ok hiszen a _Hast.Samples.SampleAssembly_-ban van.